src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: " + getArea()
+                + " radius: " + radius + " color " + color);

```

Comment:
The `color` field is protected in the `Figure` class, which means it can be accessed directly in the `Circle` class. Ensure that the `color` field is correctly accessed in the `draw()` method. Currently, it seems like there might be an issue with how `color` is being used in the `System.out.println()` statement.